Autopilot/PathManager/Inc/pathManager.hpp

```diff
@@ -1,5 +1,6 @@
 #pragma once
 #include "pathManagerStateManager.hpp"
+#include "landingTakeoffManager.hpp"
```

Comment:
I think this file is not used in the header, right ? If that's the case, just stick it in the appropriate .cpp instead

Reply:
the _FlightStage struct is used here, but i'm probably gonna move it to pathDatatypes like dhruv said.